packages/api-explorer/src/components/SelectorContainer/ApiSpecSelector.tsx

```diff
@@ -49,8 +49,8 @@ export const ApiSpecSelector: FC<ApiSpecSelectorProps> = ({
     description: spec.status,
   }))
 
-  const handleChange = (specKey: string) => {
-    specDispatch(selectSpec(specs, specKey))
+  const handleChange = async (specKey: string) => {
```

Comment:
Is the async needed?

Reply:
That's something you added on 7/23/21, probably when working on the fetching spec on demand. Going to leave it in until you have more feedback.

Reply:
It can removed. I vaguely remember that I had logic to make an aync call here but ended up moving that code to useEffect elsewhere. Forgot to remove the async